docs/tutorials/coord_sys_tutorial.md

```diff
@@ -0,0 +1,211 @@
+# Coordinate System Tutorial
+
+## Overview
+
+MMDetection3D uses three different coordinate systems. The existence of different coordinate systems in the society of 3D object detection is necessary, because for various 3D data collection devices, such as LiDAR, depth camera, etc., the coordinate systems are not consistent, and different 3D datasets also follow different data formats. Early works, such as SECOND, VoteNet, convert the raw data to another format, forming conventions that some later works also follow, making the conversion between coordinate systems even more complicated.
+
+Despite the variety of datasets and equipments, by summarizing the line of works on 3D object detection we can roughly categorize coordinate systems into three:
+
+- Camera coordinate system -- the coordinate system of most cameras, in which the positive direction of the y-axis points to the ground, the positive direction of the x-axis points to the right, and the positive direction of the z-axis points to the front.
+- LiDAR coordinate system -- the coordinate systems of many LiDARs, in which the negative direction of the z-axis points to the ground, the positive direction of the x-axis points to the front, and the positive direction of the z-axis points to the left.
+- Depth coordinate system, the coordinate systems used by VoteNet, H3DNet, etc., in which the negative direction of the z-axis points to the ground, the positive direction of the x-axis points to the right, and the positive direction of the z-axis points to the front.
```

Comment:
1. I think `the coordinate systems` should be `the coordinate system`.
2. There is confusion why the negative direction of the z-axis to the ground but the positive direction of the z-axis to the front. I remember that the front is the positive direction of the y-axis.